sceval_frontend/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # base image
-FROM node:latest as build-stage
+FROM node:12.9.0-alpine as build-stage
```

Comment:
Writing a tag name is important I think. Because `latest` will be changed when the latest version will go up.
So it may be fail to build.

Reply:
> `index.html` that is built by `create-react-app` should exists for production environment.
> Nginx will serve this file at first.
> 
> And I refactor a little of related docker.

@reprimande I see. Thanks for looking into this. I'll try to re-create DI with the different tag and  removing the `rm build/index.html`.